pubspec.lock

```diff
@@ -42,7 +42,7 @@ packages:
       name: async
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.8.1"
+    version: "2.6.1"
```

Comment:
Upgrade your dart and flutter SDK. We should not be using previous versions of packages.